.github/CONTRIBUTING.md

```diff
@@ -1,65 +1,55 @@
+* **Already fixed**? Check whether your issue/feature is already fixed/implemented.
+* **Still relevant**? Check if the issue still exists in the latest release/beta version.
+* **Can you fix it**? If you are an Android/Java developer, you are always welcome to fix an issue or implement a feature yourself. PRs welcome! See [Code contribution](#code-contribution) for more info.
+* **Is it in English**? Issues in other languages will be ignored unless someone translates them.
+* **Is it one issue**? Multiple issues require multiple reports, that can be linked to track their statuses.
+* **The template** Fill it out, everyone wins. Your issue has a chance of getting fixed.
 
-## Bug Fixing
-* If you want to help NewPipe to become free of bugs (this is our utopic goal for NewPipe), you can send us an email to
-<a href="mailto:tnp@newpipe.schabi.org">tnp@newpipe.schabi.org</a> to let us know that you intend to help. We'll send you further instructions. You may, on request,
-register at our [Sentry](https://sentry.schabi.org) instance (see section "Crash reporting" for more information).
+## Join the developer team
+* Help rid NewPipe of bugs by sending an e-mail saying you want to join to <a href="mailto:tnp@newpipe.schabi.org">tnp@newpipe.schabi.org</a>, mentioning what you want to help with.
+You may be asked to register at our [Sentry](https://sentry.schabi.org) instance.
 
 ## Translation
 
-* NewPipe is translated via [Weblate](https://hosted.weblate.org/projects/newpipe/strings/). You can log in there
-with your GitHub account.
-* If the language you want to translate is not on Weblate, you can add it: see [How to add a new language](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-add-a-new-language-to-NewPipe) in the wiki.
+* NewPipe is translated via [Weblate](https://hosted.weblate.org/projects/newpipe/strings/).
+Log in there with your GitHub account, or register.
+* Add the language you want to translate if it is not there already: see [How to add a new language](https://github.com/TeamNewPipe/NewPipe/wiki/How-to-add-a-new-language-to-NewPipe) in the wiki.
 
 ## Code contribution
 
-* If you want to add a feature or change one, please open an issue describing your change. This gives the team and community a chance to give feedback before you spend any time on something that could be done differently or not done at all. It also prevents two contributors from working on the same thing and one being disappointed when only one user's code can be added.
-* Stick to NewPipe's style conventions: follow [checkStyle](https://github.com/checkstyle/checkstyle). It will run each time you build the project.
-* Do not bring non-free software (e.g. binary blobs) into the project. Also, make sure you do not introduce Google
+* Add a feature or change one by opening an issue describing your change. This gives the team and community a chance to give feedback before you spend time on something already underway,what shhould be done differently, or avoided.
```

Comment:
"something already underway" is needlessly vague. Other than that, some consistency added.
```suggestion
* Add a feature or change one by opening an issue describing your change. This gives the team and the community a chance to give feedback before you spend time on a feature that is already in development, should be done differently, or should be avoided completely.
```